lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -364,21 +358,22 @@ def start(self):
                 p["events"]["true_source_fov_offset"] = calculate_source_fov_offset(
                     p["events"], prefix="true"
                 )
-                p["events"]["reco_source_fov_offset"] = p["events"]["true_source_fov_offset"]
+                p["events"]["reco_source_fov_offset"] = p["events"][
+                    "true_source_fov_offset"
+                ]
 
         self.log.debug(p["simulation_info"])
         gammas = self.mc_particle["gamma"]["events"]
         geomag_params = self.mc_particle["gamma"]["geomag_params"]
         self.log.info(geomag_params)
+        mean_fov_offset = get_mc_fov_offset(self.mc_particle["gamma"]["file"])
```

Comment:
`mean_fov_offset` -> `mc_fov_offset` or maybe something similar

Reply:
I prefer mean, because of ring wobbles tbh.
But either way is fine